node_modules/gh-auth/lib/api.js

```diff
@@ -289,13 +423,13 @@ var contextMiddleware = function(req, res, next) {
  * @param  {String}         cookieSecret        The cookie secret to encrypt the session cookie with
  * @api private
  */
-var registerSerializeUser = function(cookieSecret) {
-    passport.serializeUser(function(authInfo, callback) {
+var _registerSerializeUser = function(cookieSecret) {
+    passport.serializeUser(function(ghAuthInfo, callback) {
         // Serialize the user id and whether the user is logged in
         // on the global admin server to the session cookie
         var sessionCookie = {
-            'userId': authInfo.user.id,
-            'isGlobalAdmin': authInfo.app.isGlobalAdmin
+            'userId': ghAuthInfo.user.id,
```

Comment:
When `*ClientPasswordAuthentication` is used, will this serializer run? And if it does, will `ghAuthInfo.user` be undefined?